packages/components/src/entities/SampleAliquotDetailHeader.tsx

```diff
@@ -9,19 +9,21 @@ import { DefaultRenderer } from '../internal/renderers/DefaultRenderer';
 import { SAMPLE_STATE_COLUMN_NAME } from '../internal/components/samples/constants';
 import { SampleStatusTag } from '../internal/components/samples/SampleStatusTag';
 import { getSampleStatus } from '../internal/components/samples/utils';
+import {UserDetailsRenderer} from "../internal/renderers/UserDetailsRenderer";
```

Comment:
Note: I have not run the linter on this PR yet. I will do that after CR is complete.